reference.md

```diff
@@ -12,15 +12,16 @@ various APIs, CLIs, and file formats.
 |:--------------------------------------------------------------------|:----------------------------------------------------------------|
 | [Dockerfile](/engine/reference/builder/)                            | Defines the contents and startup behavior of a single container |
 | [Compose file](/compose/compose-file/)                              | Defines a multi-container application                           |
+| [Daemon CLI (dockerd)]()| 
```

Comment:
This doesn't lead anywhere. It also needs a description.

You could also delete it since it is repeated below and I'm not sure Daemon CLI is a file format.

Reply:
see I knew there was a mistake in the reference. :) Thank you for catching, it's removed.